apps/google-analytics-4/frontend/src/components/config-screen/HomeAnalyticsPage.tsx

```diff
@@ -0,0 +1,188 @@
+import { useCallback, useState, useEffect } from 'react';
+import { AppExtensionSDK } from '@contentful/app-sdk';
+import { Box } from '@contentful/f36-components';
+import GoogleAnalyticsIcon from 'components/common/GoogleAnalyticsIcon'; import omitBy from 'lodash/omitBy';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { styles } from 'components/config-screen/HomeAnalytics.styles'
+import Splitter from 'components/common/Splitter';
+import ApiAccessPage from 'components/config-screen/api-access/ApiAccessPage';
+import ConfigurationPage from 'components/config-screen/configuration/ConfigurationPage';
+import AboutSection from 'components/config-screen/header/AboutSection';
+import { AppInstallationParameters, ServiceAccountKey, ServiceAccountKeyId } from 'types';
+import { convertServiceAccountKeyToServiceAccountKeyId, convertKeyFileToServiceAccountKey, AssertionError } from 'utils/serviceAccountKey';
```

Comment:
Common Pattern Comment: Use absolute importing instead of relative (Credit: Lillian Bitner)